en/css/README.md

```diff
@@ -191,7 +191,7 @@ Go ahead and name some parts of the HTML code. Add a class called `page-header`
 </div>
 ```
 
-And now add a class `post` to your `div` containing a blog post.
+And now add a class `post` to your `div` that displays each blog post details.
```

Comment:
This seems grammatically wrong. It also might be semantically confusing, as this is the view for an overview over all published blog posts, not the view to show a post's _details_. (Though, obviously, _some_ details are shown in the overview, too.)